week-4/A-objects-intro/exercise-part-0.js

```diff
@@ -4,4 +4,11 @@ Describe your own laptop as a JavaScript object
 
 Try to think of as many properties as you can!
 

```

Comment:
Good.  `ahmetLaptop` would be more correct for variable name capitalisation, under most style guides.  Rule number 1 is be consistent.